kaguna/dimba-api
# #161416054 Dockerize application

#### What does this PR do?
Dockerizes this API

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
Ensure that you have [docker](https://docs.docker.com/install/) and [compose](https://docs.docker.com/compose/install/#install-compose) installed before proceeding.
 - Open `application.yml` and update accordingly as shown below
  ```
  POSTGRES_USER: postgres
  POSTGRES_HOST: db
   ```
- On your terminal run: `docker-compose build`
- You also need to create the databases. In another terminal, run:
  `docker-compose run web rake:db create`
- Finally start up application `docker-compose up` and visit [http://localhost:3000/](http://localhost:3000/)

#### PT story
[#161416054](https://www.pivotaltracker.com/story/show/161416054)

#### Screenshots
N/A
